docs/tasks/debug-application-cluster/audit.md

```diff
@@ -72,6 +72,9 @@ In both cases, audit events structure is defined by the API in the
 `audit.k8s.io` API group. The current version of the API is
 [`v1beta1`][auditing-api].
 
+**Note:** in case of patches, request objects are arrays, containing operations, not the appropriate
```

Comment:
Maybe:

> Audit events for PATCH requests are an array of request objects containing operations, not the entire API object.

what do you think?

Reply:
That's a limiting statement. Who knows what types of request will support using JSON arrays as a request body in the future? Rewrote the paragraph though